packages/connect-examples/browser-inline-script/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>@trezor/connect example</title>
+    </head>
+    <body>
+        <script type="text/javascript" src="https://connect.trezor.io/9/trezor-connect.js"></script>
+        <script type="text/javascript">
+            window.TrezorConnect.init({
+                lazyLoad: true,
+                manifest: {
+                    email: 'developer@xyz.com',
+                    appUrl: 'http://your.application.com',
+                },
+            });
+
+            trezorGetAddress = function () {
+                console.log('init');
```

Comment:
nit: rm console?

Reply:
Perhaps something like this makes sense for example purposes? I'd change it to `console.info()` though.

Reply:
maybe it makes sense to log the response (below) but this one doesn't imho